keyboards/wilba_tech/zeal65/rules.mk

```diff
@@ -20,7 +20,7 @@ COMMAND_ENABLE = no        # Commands for debug and configuration
 NKRO_ENABLE = yes           # Enable N-Key Rollover
 BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
 AUDIO_ENABLE = no           # Audio output
-RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight. 
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.
```

Comment:
```suggestion
RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight. 
```

Reply:
Why leave the trailing white space?

Reply:
That can be fixed in another PR. Try to keep keyboard changes separate from keymap/userspace changes.

Reply:
OK